src/lib/logger/vercel/vercelLogger.test.ts

```
@@ -0,0 +1,707 @@
+						'Predict the top 3 largest city by 2050. For each, return the name, the country, the reason why it will on the list, and the estimated population in millions.',
+					providerOptions: {
+						maxim: {
+							traceId: traceId,
+							spanId: spanId,
+						}
+					}
+				});
+
+				const { object } = await generateObject({
+					model: model,
+					prompt: 'Extract the desired information from this text: \n' + rawOutput,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)